participants/NE-Intelligent-Drive.md

```diff
@@ -0,0 +1,76 @@
+## **Project Name and Team Slack Handles**
+
++ **Name:** NE Intelligent Drive
+
++ **Slack:**  JunyaoRen (Display Name: Josh)
+
++ **Member ID:** UJ9UF1LEP
+
+## **(1) Brief Project Description**
+
+- We have developed this drive based on IPFS in 2018, and now we are connecting it to the Filecoin network to complete its business model.
+
+## **(2) Link to Project UI**
+
+- [NE Intelligent Drive](https://www.nonentropy.com/fildiscdownload.html)
+
+## **(3) What does your application/UI do?**
+
+- Users can consume their points to upload and download files with this drive.
+- Users can share their files to other users with file names or file hashes.
+- Users can choose between IPFS and Filecoin networks to store their files.
+
+## **(4) If your project is using a curated dataset, which one are you using?**
+
+- N/A
+
+## **(5) If your project is not using a curated dataset, please tell us a little bit more about your data by answering the questions below.**
+
+**What sorts of data will you collect?**
+
+- We are not collecting any data by ourselves, but instead our users will utilize this tool to store their personal files, which can be either family photos, videos, documents, or anything they would like to keep for later use. 
+
+**Do you own or have exclusive rights to the data you plan to use?**
+
+- All the right of the data in our application belongs to our users. 
+
+**Is the data license available online?**
+
+- N/A
+
+**Does the data contain any sensitive or illegal information?**
+
+- We will not put any censorship on users' data on our drive, nor will be check them for sensitive or illegal information.  Also as part of our users will choose to use our supported encryption tool, we would never know what are in the files. 
+
+## **(6) How much data are you planning to store to the Filecoin network during the Slingshot competition?**
+
+- 1PB
+
+## **(7) How are you structuring the data?**
+
+- On our end, the data information (including data CID's and filenames) is associated with each user in our database, so that users could very easily retrieve their data. 
+
+## **(8) What pre-processing are you doing before ingesting the data?**
+
+- Each file that uploaded by user will be encrypted(optionally) before pushed to IPFS or Filecoin. 
+
+## **(9) What tech stack will you use to store the data?**
+
+- Lotus		js-ipfs		Electron 		Vue.js		iView		Beego		Node.js		MySQL		NeDB		JDK 11		Redis
```

Comment:
You may want to use Powergate as a hot storage option when storing data for easy retrieval.